### ISSUE TITLE
Google Analytics 4 config: no need for anonymize_ip parameter

### ISSUE THREAD

**Issue by chalin** [gohugoio/hugo#10093]
As mentioned in https://support.google.com/analytics/answer/9019185#IP:

> In Google Analytics 4, **IP anonymization is not necessary** since IP addresses are not logged or stored.

Consider the GA internal template:

https://github.com/gohugoio/hugo/blob/9c24b86e4b634d7db761a3e57c1b64fead880aff/tpl/tplimpl/embedded/templates/google_analytics.html#L3-L13

At line 11, `anonymize_ip` is being passed as a configuration parameter unnecessarily.

Hugo version 0.101.0

**Comment by coliff** [comment]
Agreed. `anonymize_ip` is on by default (and can't be disabled at all) in GA4 and as Universal Analytics is deprecated then this feature should be deprecated too.

**Comment by StefMa** [comment]
Since the [template is deprecated](https://github.com/gohugoio/hugo/issues/10899), this issue can be closed as obsolete, right?

**Comment by coliff** [comment]
> Since the [template is deprecated](https://github.com/gohugoio/hugo/issues/10899), this issue can be closed as obsolete, right?

No, The current Google Analytics code in Hugo still includes `anonymize_ip` parameter but should be removed.

You can see this if you inspect code of: https://gohugo.io (line 13).
There is no need for `anonymize_ip:!0` because all traffic is always anonymized and it can't be changed.

**Comment by StefMa** [comment]
Thanks @coliff.

It seems I was a bit confused with the `_async` version of Google Analytics 👍
